README.md

```diff
@@ -103,6 +103,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Dropcode](https://github.com/egoist/dropcode) - Simple and lightweight code snippet manager.
 - [JournalV](https://github.com/ahmedkapro/journalv) - Journaling app for your days and dreams.
 - [GitBar](https://github.com/mikaelkristiansson/gitbar) - System tray app for GitHub reviews.
+- [Wirefish](https://github.com/stefanodevenuto/wirefish) - Multiplatform packet sniffer and analyzer
```

Comment:
```suggestion
- [Wirefish](https://github.com/stefanodevenuto/wirefish) - Cross-platform packet sniffer and analyzer.
```

Reply:
Cross-platform is the word that we used for other apps in the list too, I'd make sense to keep the lingo the same.

Reply:
Sure! I'll fix it